src/components/Modal/node/FundChannelModal.tsx

```diff
@@ -31,7 +31,7 @@ export const FundChannelModal = ({ ...props }: FundChannelModalModalProps) => {
   const canFund = !(!amount || parseFloat(amount) <= 0 || !channelId);
 
   useEffect(() => {
-   window.addEventListener('keydown', handleEnter as EventListener);
+    window.addEventListener('keydown', handleEnter as EventListener);
```

Comment:
I believe we talked about this.
Generally it would be best to let the browser handle enter key input by presenting an html form and handling the submit event.
But I understand that this might be a tough refactor so I just want to raise awareness for the issue.

I would be happy if the enter key starts working on all input modals, e.g. fund channel and so on.. :)

Reply:
Creating issues